media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
```

Comment:
Then again, any reason we cannot just use this? <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css"> @craigcook @alexgibson